tests/omrabbitmq-injector_via_improg.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
just to keep test complexity low: can't we use the regular methods (via tcpflood or injectmsg) to inject messages? Having improg run here considerably increases the tested set and so may lead to failures from other components. Just a thought...

Reply:
This is a very good notice. Until, now the problem is not there. I will simplify the inject as soon as a get a stable way of testing output. Again docker-ubuntu18-san is a "punishment" for me. I have not yet understand what's wrong in my tests infra. Do I have a way to detect this particular configuration to skip some tests on if I am not able to find why I get problems there.

Reply:
I used improg just because I knew it.

Reply:
> Do I have a way to detect this particular configuration to skip some tests on if I am not able to find why I get problems there.

You can skip a platform, but you cannot skip a specific test run - after all, that means the test fails on that platform (here: ubuntu18). That it works in some configurations doesn't help. So we need to find out what the issue really is. Let's do that on the main discussion thread, not this review comment.